packages/design-system/src/css/_tokens.dark.scss

```diff
@@ -36,6 +36,9 @@
 	--color-canvas-selected: var(--prim-gray-0-alpha-025);
 	--node-type-main-color: var(--prim-gray-420);
 
+	// Sticky
+	--color-sticky-code-background: var(--color-background-light);
```

Comment:
The contrast on this is not good.
![image](https://github.com/n8n-io/n8n/assets/196144/e7ba655f-5456-4cd9-9842-58db2a23b44a)

Not sure `--color-secondary` needs to be brighter in dark mode, or if we need to define a new `--color-sticky-code-foreground` variable.

Reply:
ya it's not perfect.. but unblock this